File: skills/quality/dry-test/fixtures/swift/clone.swift
func alpha(value: Int) -> String {
    // Only alpha carries a comment. Comments are spelling, so beta must still match it.
    let label = "alpha: "
    let total = value + 1
    if total > 10 {
        return label + String(total)
    }
    return label
}

func beta(number: Int) -> String {
    let tag = "beta: "
    let amount = number + 2
    if amount > 20 {
        return tag + String(amount)
    }
    return tag
}

File: skills/quality/dry-test/fixtures/csharp/Clone.cs
class Clone
{
    string Alpha(int value)
    {
        // Only Alpha carries a comment. Comments are spelling, so Beta must still match it.
        string label = "alpha: ";
        int total = value + 1;
        if (total > 10)
        {
            return label + total.ToString();
        }
        return label;
    }

    string Beta(int number)
    {
        string tag = "beta: ";
        int amount = number + 2;
        if (amount > 20)
        {
            return tag + amount.ToString();
        }
        return tag;
    }
}
